.github/renovate.json5

```diff
@@ -52,6 +52,13 @@
       matchPackagePatterns: ["strum"],
       description: "Weekly update of strum dependencies",
     },
+    {
+      groupName: "ESLint",
+      matchManagers: ["npm"],
+      matchPackagePatterns: ["eslint"],
```

Comment:
I'm not sure what the syntax is here but I was wondering if this will restrict other package which contains "eslint" in it like "typescript-eslint". I assume not.